tox.ini

```diff
@@ -1,14 +1,31 @@
 [tox]
-envlist = py{27,36},django{111}
+envlist = py{27,35}-django{111},py{35,36}-django{20,21,22},quality
 skip_missing_interpreters = True
 
 [testenv]
+deps =
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
+    -r{toxinidir}/requirements/django.txt
+commands = pytest --disable-pytest-warnings --nomigrations {posargs}
+
+
+[testenv:pytest-py27]
```

Comment:
Great, I think the last thing is to run these two new tox environments in Travis in addition to what's already there.  That should catch any regressions in the ability to use `opaque-keys` without Django installed.